src/ngx_http_lua_exten.h

```diff
@@ -0,0 +1,20 @@
+
+/*
+ * Copyright (C) Yichun Zhang (agentzh)
+ * Copyright (C) hongliang5316
```

Comment:
To get this merged, you have to grant your copyright to the authors of this module.